master/internal/api_tasks.go

```diff
@@ -144,6 +144,38 @@ func (a *apiServer) canGetTaskAcceleration(ctx context.Context, taskID string) e
 	return nil
 }
 
+func (a *apiServer) canGetAllocation(ctx context.Context, allocationID string) error {
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return err
+	}
+
+	if !strings.Contains(allocationID, ".") {
```

Comment:
if this always has to be true I'd add it to `model.Allocation` as a Validate method.

Reply:
I'd like to have more robust validation check than only checking if the string contains a period, I'll create a backlog ticket to follow up on this